Yoast/yoast-components
# Automatically open relevant collapsible

## Summary

This PR can be summarized in the following changelog entry:

* Expands the AnalysisCollapsible(s) that are currently the most urgent in the content analysis. For example: if there are no problems, but there are results in the categories 'needs improvement' and 'good', the 'needs improvement' AnalysisCollapsible is expanded by default. 

## Test instructions

This PR can be tested by following these steps:

* Link this branch to your local Wordpress environment with the wordpress-seo release/5.9 branch. 
* Play around with empty result categories. The most urgent category should always be expanded. The order is: errors, problems, needs improvements, considerations, good. For example: Write a post with no 'problems', but with 'needs improvements'. Make sure 'needs improvements' is expanded by default.
* **Not fixed in this PR:** Write a post with one 'problem' and one or more 'needs improvement's. Solve the problem. The 'needs improvement' should expand (and the 'problem' should disappear, of course).

Partial fix of: https://github.com/Yoast/wordpress-seo/issues/8392. See comment on that issue for more context.
